zerver/webhooks/github/view.py

```diff
@@ -207,8 +207,9 @@ def get_push_commits_body(payload: Dict[str, Any]) -> str:
     )
 
 def get_public_body(payload: Dict[str, Any]) -> str:
-    return "{} made [the repository]({}) public.".format(
+    return "{} made [the repository `{}`]({}) public.".format(
```

Comment:
We're hyperlinking too much text here. Instead of
```
[the repository `{}`]({})
```
Let's go with:
```
the repository [{}]({})
```

For example, "the repository [bytecodealliance/wasmtime](https://github.com/bytecodealliance/wasmtime)" looks better than "[the repository bytecodealliance/wasmtime](https://github.com/bytecodealliance/wasmtime)".

Reply:
Makes sense, and I changed things accordingly.